include/exadg/time_integration/time_int_multistep_base.h

```diff
@@ -176,20 +169,20 @@ class TimeIntBDFBase : public TimeIntBase
    * Initializes the solution vectors by prescribing initial conditions or reading data from
    * restart files and initializes the time step size.
    */
-  virtual void
-  initialize_solution_and_time_step_size(bool do_restart);
+  void
+  initialize_vectors_and_time_step_size(bool do_restart);
 
   /*
-   * Initializes the solution vectors at time t
+   * Initializes vectors needed for the multistep scheme
    */
   virtual void
-  initialize_current_solution() = 0;
+  initialize_multistep_dof_vectors() = 0;
```

Comment:
The problem here is, that in Adams methods, we do not need to keep track of former solutions. Instead, we have to keep track of evaluated operators. I still think all multistep methods should be derived from the same base class, since the general principle of those methods are the same.

Reply:
With the explanations you gave in https://github.com/exadg/exadg/pull/261#issuecomment-1468530690, I would be happy if we could insert `former` and use `initialize_former_multistep_dof_vectors()`.

Reply:
Regarding the inheritance discussion, I think it could make sense to have both functions in the base class for multistep methods, supposing that we find intuitive names.